src/app/[locale]/dashboard/categories/[id]/page.tsx

```
@@ -0,0 +1,22 @@
+import { useTranslations } from "next-intl";
+
+interface PageProps {
+    params: { id: string };
+}
+
+export default async function UpdateCategoryPage({ params }: PageProps) {
+    const { id } = params;
+
+    const data: CategoryResponse = await getSpacificCategory(id);
+    console.log(data?.category);
+    const { name, image } = data?.category;
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)